OWML.ModHelper.Interaction/InterfaceProxyFactory.cs

```diff
@@ -9,29 +9,33 @@ namespace OWML.ModHelper.Interaction
 {
     public class InterfaceProxyFactory
     {
-        private readonly ModuleBuilder ModuleBuilder;
-
-        private readonly IDictionary<string, InterfaceProxyBuilder> Builders = new Dictionary<string, InterfaceProxyBuilder>();
+        private readonly ModuleBuilder _moduleBuilder;
+        private readonly IDictionary<string, InterfaceProxyBuilder> _builders = new Dictionary<string, InterfaceProxyBuilder>();
 
         public InterfaceProxyFactory()
         {
-            var assemblyBuilder = AppDomain.CurrentDomain.DefineDynamicAssembly(new AssemblyName($"OWMLInteraction.Proxies, Version={this.GetType().Assembly.GetName().Version}, Culture=neutral"), AssemblyBuilderAccess.Run);
-            this.ModuleBuilder = assemblyBuilder.DefineDynamicModule("OWMLInteraction.Proxies");
+            var assemblyName = new AssemblyName($"OWMLInteraction.Proxies, Version={GetType().Assembly.GetName().Version}, Culture=neutral");
+            var assemblyBuilder = AppDomain.CurrentDomain.DefineDynamicAssembly(assemblyName, AssemblyBuilderAccess.Run);
+            _moduleBuilder = assemblyBuilder.DefineDynamicModule("OWMLInteraction.Proxies");
         }
 
-        public TInterface CreateProxy<TInterface>(object instance, string sourceModID, string targetModID) where TInterface : class
+        public TInterface CreateProxy<TInterface>(object instance, string sourceModId, string targetModId) where TInterface : class
         {
             if (instance == null)
+            {
                 throw new InvalidOperationException("Can't proxy access to a null API.");
+            }
             if (!typeof(TInterface).IsInterface)
+            {
                 throw new InvalidOperationException("The proxy type must be an interface, not a class.");
+            }
 
             var targetType = instance.GetType();
-            var proxyTypeName = $"OWMLInteraction.Proxies.From<{sourceModID}_{typeof(TInterface).FullName}>_To<{targetModID}_{targetType.FullName}>";
-            if (!this.Builders.TryGetValue(proxyTypeName, out InterfaceProxyBuilder builder))
+            var proxyTypeName = $"OWMLInteraction.Proxies.From<{sourceModId}_{typeof(TInterface).FullName}>_To<{targetModId}_{targetType.FullName}>";
```

Comment:
Maybe change these to ModName, since ID isn't used anywhere else?

Reply:
you mean just remove the modID part?

Reply:
yeah, sourceMod and targetMod could work. i was thinking sourceModName and targetModName (since these are actually the mod's uniquenames)

Reply:
perfect :)
you wanna do it should i?

Reply:
you can do it, i'm guessing you already have the branch on your pc :p

Reply:
how 'bout now?

Reply:
or now? :P